src/App.scss

```diff
@@ -49,3 +49,11 @@
 .add-100vh {
   height: 100vh
 }
+
+.force-text-black {
+  color: var(--pf-c-dropdown__menu-item--Color) !important;
```

Comment:
It seems like `!important` is not necessary here; without it I still see the link black.
And while you are at it, can you delete these lines:
```css
.force-padding-left {
  padding-left: 1rem !important;
}
```

no component is using this abomination :vomiting_face:  anymore